apis/apiextensions/v1alpha1/revision_types.go

```diff
@@ -272,6 +274,7 @@ type PatchPolicy struct {
 	// +optional
 	// +immutable
 	FromFieldPath *FromFieldPathPolicy `json:"fromFieldPath,omitempty"`
+	MergeOptions  *MergeOptionsPolicy  `json:"mergeOptions,omitempty"`
```

Comment:
```suggestion
	MergeOptions  *xpv1.MergeOptions  `json:"mergeOptions,omitempty"`
```

Would this work? I believe it does the same thing, but more closely mirrors the implementation in the `Composition` type.

Reply:
@negz i have remove the type casting 
is this what you was thinking about?